test/customValidators_test.js

```diff
@@ -300,7 +300,65 @@ describe('validateChannelParams()', function() {
     }`;
     const parsedInput = JSON.parse(inputString);
 
-    expect(() => validateChannelParams(parsedInput, inputString, input)).to.throw('Not all channel parameters are described with parameter object');
+    expect(() => validateChannels(parsedInput, inputString, input)).to.throw('Not all channel parameters are described with parameter object');
+  });
+
+  it('should successfully validate channel name without variable', async function() {
+    const inputString = `{
+      "channels": {
+        "test/test01": {
+          "parameters": {
```

Comment:
you are validating here a channel name, so instead of providing a parameter here, I think you could just pass empty object